packages/gridstack/src/components/SaveDashboardModal/SaveDashboardModal.vue

```diff
@@ -53,4 +60,7 @@ const submitAndClose = () => {
 .header2 {
   min-width: 300px;
 }
+.error-message {
+  color: red;
```

Comment:
We should use the color variable here. I think something like `var(--color-danger-xxx)`. See color guide in docs https://lion5.github.io/component-library/?path=/docs/guide-colors--docs
Otherwise, the error message is not themed.

Reply:
Not needed anymore as now using ErrorBox instead